ctrlf.el

```diff
@@ -183,29 +253,34 @@ mess."
 
 (cl-defun ctrlf--search
     (query &key
-           (regexp :unset) (backward :unset)
-           (literal :unset) (forward :unset)
+           (translator :unset) (style :unset)
```

Comment:
Why does this need both `translator` and `style` as arguments? Can't the former be derived from the latter?

Reply:
Yes, `style` is enough. I just did this so that `ctrlf--search` can be used completely out of the environment, including the variable `ctrlf-style-alist`. I can remove it if you think it's not needed.

Reply:
Done. I put this change in a separate commit, you can pick or drop it as you like.